app/src/main/java/com/github/se/cyrcle/ui/authentication/AuthenticatorImpl.kt

```diff
@@ -91,9 +93,7 @@ class AuthenticatorImpl @Inject constructor(private val auth: FirebaseAuth) : Au
 
           val user =
               User(
-                  userId = authResult.user?.uid ?: "",
-                  username = authResult.user?.displayName ?: "",
-                  email = authResult.user?.email ?: "")
+                  UserPublic(authResult.user!!.uid, authResult.user!!.displayName!!), UserDetails())
```

Comment:
Why did you removed the email argument here ? That's just to understand

Reply:
That is a big mistake on my part, thanks for spotting it